src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVaultCertificateContact.cs

```diff
@@ -56,63 +48,67 @@ public class RemoveAzureKeyVaultCertificateContact : KeyVaultCmdletBase
                    ValueFromPipelineByPropertyName = true,
                    HelpMessage = "Specifies the email address of the contact.")]
         [ValidateNotNullOrEmpty]
-        public string EmailAddress { get; set; }
+        public string[] EmailAddress { get; set; }
 
         /// <summary>
         /// PassThru parameter
         /// </summary>
-        [Parameter(HelpMessage = "This cmdlet does not return an object by default. If this switch is specified, it returns the contact object.")]
+        [Parameter(Mandatory = false,
+                   HelpMessage = "This cmdlet does not return an object by default. If this switch is specified, it returns the contact object.")]
         public SwitchParameter PassThru { get; set; }
 
-        protected override void ProcessRecord()
+        public override void ExecuteCmdlet()
         {
-            if (ShouldProcess(EmailAddress, Properties.Resources.RemoveCertificateContact))
+            foreach (string email in EmailAddress)
```

Comment:
Same observation as in the matching AddCertContact cmdlet - consider storing the input email addresses in a hashset, making a single call to the service to retrieve the contacts, and then updating the contacts in a single pass.